README.md

```diff
@@ -0,0 +1,9 @@
+# HPC Team Pre-Commit Hooks
+
+Hooks for pre-commit used by the HPC Team of the Blue Brain Project.
```

Comment:
The description is a bit weak, but I honestly do not understand what this repo is about.

Pinging @tristan0x for clarifications.

Reply:
I tried to clarify what this repo is about.

Reply:
Thank you, @tristan0x and @matz-e!